src/schema/objects/columns.yaml

```diff
@@ -173,29 +193,34 @@ low_cutoff:
     - n/a
 manufacturer:
   name: manufacturer
+  display_name: Manufacturer
   description: |
     The manufacturer for each electrode.
     Can be used if electrodes were manufactured by more than one company.
   type: string
 mapping:
   name: mapping
+  display_name: Label mapping
   description: |
     Corresponding integer label in the standard BIDS label lookup.
   type: integer
 material:
   name: material
+  display_name: Electrode material
```

Comment:
probably unrelated to this PR: all of the above examples adding "Participant", "Electrode" and other qualifiers suggest eventual possible collision. E.g. couldn't material be eventually be a legit column for some kind of "injector" or "sample"... dunno.  I see many `id`s below include their "domain".  i.e. here to have `electrode_material` ?